src/util/saveArticles.ts

```diff
@@ -260,21 +242,27 @@ export async function saveArticles(zimCreator: ZimCreator, downloader: Downloade
   const rendererBuilder = new RendererBuilder()
 
   let rendererBuilderOptions: RendererBuilderOptions
+
+  let mainPageRenderer
+  let articlesRenderer
   if (forceRender) {
     rendererBuilderOptions = {
       renderType: 'specific',
       renderName: forceRender,
     }
+    // All articles and main page will use the same renderer if 'forceRender' is specified
+    mainPageRenderer = await rendererBuilder.createRenderer(rendererBuilderOptions)
+    articlesRenderer = await rendererBuilder.createRenderer(rendererBuilderOptions)
   } else {
     rendererBuilderOptions = {
-      renderType: 'auto',
+      renderType: 'desktop',
     }
+    mainPageRenderer = await rendererBuilder.createRenderer(rendererBuilderOptions)
+    // If the mobile renderer API is not available, switch articles rendering to the auto mode instead
+    rendererBuilderOptions.renderType = hasWikimediaMobileRestApi ? 'mobile' : 'auto'
+    articlesRenderer = await rendererBuilder.createRenderer(rendererBuilderOptions)
```

Comment:
Do simpler, just call `createRenderer` with the appropriate value or expression. Variable should be instanciated only when:
* modified at some point
* use more than once

... ortherwise this is just a wast of resources.

Reply:
Done, pls check.